back/engines/commercial/multi_tenancy/app/services/multi_tenancy/templates/tenant_serializer.rb

```diff
@@ -219,7 +227,10 @@ def infer_scope_class(scope)
 
       def serialize_admin_publications(scope)
         # This code will stop working when folders can contain folders
-        serialize_records(scope.order(parent_id: :desc, ordering: :asc))
+        scope = scope
+          .where(publication_type: 'ProjectFolders::Folder') # Support nested admin publications
+          .order(parent_id: :desc, ordering: :asc)
+        serialize_records scope
```

Comment:
why only _folder_ `admin_publications`?

Reply:
We're basically keeping it the same way it worked for folders before, so no nested attributes and separate serialization of the folder admin publications (whereas for projects, the admin publication is specified through the nested attributes and needs to be filtered out here).

Ideally, I would have like the same approach for projects and folders (both with nested attributes), but supporting references between nested attributes was really getting too complicated.